data/core/get_started.md

```diff
@@ -0,0 +1,175 @@
+## story_001
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "dialogflow"}
+  - utter_switch_dialogflow
+
+## story_002
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "luis"}
+  - utter_switch_luis
+
+## story_003
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
```

Comment:
this story doesn't make much sense 😄

Reply:
this covers the case where someone says something like "I am switching from x" where x is some random product. Doesn't it make sense to just push them to the nlu quickstart in that case?

Reply:
hmmm, i mean they could also say something like "I'm switching", in which case we'd say "From what service?", and then maybe after that send the NLU quickstart link?

Reply:
I'd prefer to keep it simple for now and leave out the extra qeustion. maybe just sending both luis and dialogflow links?